tests/e2e/test_chaperone.js

```diff
@@ -96,51 +96,13 @@ const getHostAgentKey = async (serviceClient) => {
     encoded: Codec.AgentId.encode(agentPubKey)
   }
 }
-// Register test app in hha  (in real word scenario - will be done when provider registers app in hha):
-const registerTestAppInHha = async (hostedClient) => {
-  const hhaAppInfo = await hostedClient.appInfo({
-    installed_app_id: HHA_INSTALLED_APP_ID
-  });
-  const hhaCellId = hhaAppInfo.cell_data[0][0];
-
-  const happBundle = {
-    hosted_url: "https://testapp.com",
-    happ_alias: "test-app",
-    ui_path: "/path/to/test_app_ui.zip",
-    name: "Test App Hosted On Web",
-    dnas: [{
-      hash: "hC0k...",
-      path: envoyOpts.hosted_app.dnas[0].path,
-      nick: envoyOpts.hosted_app.dnas[0].nick,
-    }]
-  };
-
-  let happRegistrationId;
-  try {
-    ({
-      happ_id: happRegistrationId
-    } = await hostedClient.callZome({
-      // NOTE: Cell ID content MUST be passed in as a byte buffer not a u8int byte-array
-      cell_id: [Buffer.from(hhaCellId[0]), Buffer.from(hhaCellId[1])],
-      zome_name: 'hha',
-      fn_name: 'register_happ',
-      payload: happBundle,
-      cap: null,
-      provenance: Buffer.from(hhaCellId[1])
-    }));
-  } catch (error) {
-    throw new Error(JSON.stringify(error));
-  }
-
-  return Codec.HoloHash.encode('header', happRegistrationId);
-}
+const REGISTERED_HAPP_HASH = "uhCkkCQHxC8aG3v3qwD_5Velo1IHE1RdxEr9-tuNSK15u73m1LPOo"
```

Comment:
Why replace the function with a hardcoded value?  What about when we want to test calling a happ that is not the hha?

Reply:
Because we don't need to register a new app.  
We have to go with the assumption that the read-only instance is already installed. If we follow this pattern we would also have to install a read-only instance midway through the test.

Reply:
This would eliminate the option of testing against the mock-dna and require us to maintain the updated hash of elemental-chat every time.  In your opinion, would it be a big lift to add a flow that would install a read-only instance midway through the test? I would prob prefer to keep this flexibility.

Reply:
No, this hash can be any value. because we do not call hha with that hash anymore. 
In short that hash just has to be the same hash with which we install our read-only instance

Reply:
OK, so can we just add a note above this that mentions that this hash must match the app_name of the app in the `app-config.yml` file (which obviously should be just the hha_hash of the happ').